pom.xml

```diff
@@ -649,6 +652,44 @@
 				</plugin>
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-scm-publish-plugin</artifactId>
+					<version>1.1</version>
+					<configuration>
+						<scmBranch>gh-pages</scmBranch>
+						<tryUpdate>true</tryUpdate>
+						<serverId>github</serverId>
+						<skipDeletedFiles>true</skipDeletedFiles>
+						<!-- try to shortcut the delete process since we are not doing it anyway -->
+						<ignorePathsToDelete>
+							<ignorePathsToDelete>%regex[.*]</ignorePathsToDelete>
+						</ignorePathsToDelete>
+					</configuration>
+					<dependencies>
+						<dependency>
+							<groupId>org.apache.maven.scm</groupId>
+							<artifactId>maven-scm-api</artifactId>
+							<version>[1.9.4]</version>
```

Comment:
Why using version range instead of fixed version number ? I'm not an expert, but it seems to me that `[1.9.4]` is equivalent to just `1.9.4`. Did I miss something ?